Formula/tnftpd.rb

```diff
@@ -0,0 +1,63 @@
+class Tnftpd < Formula
+  desc "NetBSD's FTP server (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/lukemftpd/lukemftpd-51.tar.gz"
+  sha256 "969b8a35fabcc82759da2433973b9606b7b62f73527e76ac8f18d0a19f473c2a"
+
+  depends_on :xcode => :build
+
+  def install
+    system "tar zxvf tnftpd-20100324.tar.gz"
+    Dir.chdir "tnftpd-20100324"
+
+    system "./configure"
+    system "make"
+
+    File.rename "src/tnftpd", "src/ftpd"
+    File.rename "src/tnftpd.man", "src/ftpd.8"
+    File.rename "src/ftpusers.man", "src/ftpusers.5"
+    File.rename "src/ftpd.conf.man", "src/ftpd.conf.5"
+
+    bin.install "src/ftpd"
+    man8.install "src/ftpd.8"
+    man5.install "src/ftpusers.5"
+    man5.install "src/ftpd.conf.5"
+    etc.install "examples/ftpd.conf"
+    etc.install "examples/ftpusers"
+  end
+
+  def caveats
+    <<-EOS.undent
+    1. This program is installed as `ftpd', not `tnftpd'. This may or may not be
+    what you expected, depending on which systems you've used this on.
+
+    2. You may need super-user privileges to run this program properly. See the
+    man page for more details.
+
+    3. If you need this program to run automatically at login or whatever, there
+    is a plist file here, named `ftp.plist':
+    https://opensource.apple.com/tarballs/lukemftpd/lukemftpd-51.tar.gz
+
+    You can feed that into launchd by sticking it in a LaunchAgents or
+    LaunchDaemons folder. launchd may also need super-user privileges to run
+    this daemon, so you may wish to install it to /Library/LaunchAgents,
+    /Library/LaunchDaemons, /System/Library/LaunchAgents, or
+    /System/Library/LaunchDaemons.
+
+    Yet another option for auto-running this daemon is cron. You could use the
+    @reboot macro or similar.
+    EOS
+  end
+
+  test do
```

Comment:
This should just work with an `assert_match`, and again, you should use the full path, not simply "ftpd".

Reply:
@DomT4 As explained above with `telnetd`, `ftpd` doesn't seem to stick its output in the stdout. So that's why it's fiddling around with `PTY`.

I'll fix it to use the full path.

Reply:
Try the `2>&1` dance here as well.